src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int i = new Random().nextInt(Colors.values().length);
```

Comment:
make Random a class field. It will make your code more efficient - you don't need to create new Random for each method invocation - you can reuse the existing one